### DISCUSSION TITLE
Bar border not working

### DISCUSSION THREAD

**Discussion by ghost** [qtile/qtile#3117] [Q&A] [answered]
Config:
```
# Bar
screens = [
    Screen(
        top=bar.Bar(
            [
                widget.GroupBox(),
                widget.Prompt(),
                widget.Chord(
                    chords_colors={
                        'launch': ("#0000ff", "#ffffff"),
                    },
                    name_transform=lambda name: name.upper(),
                ),
                widget.Clock(format='%Y-%m-%d %a %I:%M %p'),
                widget.Systray(),
            ],
            24,
            background = "#282a36",
            border_color = "#565f89",
            border_width = [0, 0, 2, 0]
        ),
    ),
]
```

I want to add a 2px border to the bottom of my bar but it's not appearing. Am I doing something wrong? I can provide other files if needed, but this is the only bar.

**Comment by m-col** [comment, marked as answer]
This feature is so far unreleased, so is only available when running Qtile from source. Are you doing that, or are you running from a release version? If you're on a release, it won't be long until a new release is out!

**Reply by ghost** [reply]
I'm on the most recent release on Arch, so it must be unavailable. Thanks for the quick reply!
